### ISSUE TITLE
[Quest][Molten Front][Daily] Into the Fire (29206)

### ISSUE THREAD

**Issue by WhitemaneChronus** [gamefreedomgit/Maelstrom#6302]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/cata)

**Description:**

The NPC is supposed to clear the path of fire so players can follow.
Time to time he refuses to move, player cannot progress the quest.

**Database links:**

https://www.wowhead.com/cata/quest=29206/into-the-fire#comments

**Comment by Elfy279** [comment]
Okay, just experienced this bug myself on a different character that just unlocked the druids after unlocking the wardens first. Just those two unlocked for phase. When you go to the windcaller at the start of the fire, he does not move, or trigger the encounter at all. I corpse ran through the fire, and he was standing at the end as well, and when I got near him, his normal dialogue started, and the Pyrelord spawned. While the windcaller did his normal route backwards, I tried to kill the pyrelord, but it despawned before I could. I was stuck in that area again, until windcaller came back. Encounter immediately triggered, and same results. On my third attempt, I ignored the pyrelord, and followed the windcaller backwards through the fire, when almost at the start, he turned around, and started doing the path 'normally'. This time, when I got the the end, the pyrelord stayed, the druids were summoned to help fight, and the encounter worked as it should.

I wanted to add this, as I know I stated a few days ago that it was working fine, and it was for characters that had all of MF unlocked, but this char who is only 2/5 is now experiencing the issue. It is currently possible to complete, following my third trial, but definitely not working as intended

**Comment by Afgann** [comment]
> Okay, just experienced this bug myself on a different character that just unlocked the druids after unlocking the wardens first. Just those two unlocked for phase. When you go to the windcaller at the start of the fire, he does not move, or trigger the encounter at all. I corpse ran through the fire, and he was standing at the end as well, and when I got near him, his normal dialogue started, and the Pyrelord spawned. While the windcaller did his normal route backwards, I tried to kill the pyrelord, but it despawned before I could. I was stuck in that area again, until windcaller came back. Encounter immediately triggered, and same results. On my third attempt, I ignored the pyrelord, and followed the windcaller backwards through the fire, when almost at the start, he turned around, and started doing the path 'normally'. This time, when I got the the end, the pyrelord stayed, the druids were summoned to help fight, and the encounter worked as it should.
> 
> I wanted to add this, as I know I stated a few days ago that it was working fine, and it was for characters that had all of MF unlocked, but this char who is only 2/5 is now experiencing the issue. It is currently possible to complete, following my third trial, but definitely not working as intended

Do you know if the issue still happens after wednesday update?

**Comment by Elfy279** [comment]
> > Okay, just experienced this bug myself on a different character that just unlocked the druids after unlocking the wardens first. Just those two unlocked for phase. When you go to the windcaller at the start of the fire, he does not move, or trigger the encounter at all. I corpse ran through the fire, and he was standing at the end as well, and when I got near him, his normal dialogue started, and the Pyrelord spawned. While the windcaller did his normal route backwards, I tried to kill the pyrelord, but it despawned before I could. I was stuck in that area again, until windcaller came back. Encounter immediately triggered, and same results. On my third attempt, I ignored the pyrelord, and followed the windcaller backwards through the fire, when almost at the start, he turned around, and started doing the path 'normally'. This time, when I got the the end, the pyrelord stayed, the druids were summoned to help fight, and the encounter worked as it should.
> > I wanted to add this, as I know I stated a few days ago that it was working fine, and it was for characters that had all of MF unlocked, but this char who is only 2/5 is now experiencing the issue. It is currently possible to complete, following my third trial, but definitely not working as intended
> 
> Do you know if the issue still happens after wednesday update?

Yes, I experienced it just after reset today. I had to repeat the exact same process. Interestingly enough, it also happened on a 5/5 char, but has worked on others. I'm not sure what the trigger for the issue is. Maybe someone has the quest, he starts walking, but they don't kill the pyrelord so he gets stuck at the end until someone comes near, idk

**Comment by GMOuranos** [comment]
As of 6/03/2025 this quest is still not working.

**Comment by GMOuranos** [comment]
Windcaller Nordrala gets stuck at the furnace, if the player attempt fails, or he dies in the fire. The NPC needs to get reseted, so he can redo the quest.
